recipes/yamlprocessor/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "yamlprocessor" %}
+{% set version = "0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/yamlprocessor-{{ version }}.tar.gz
+  sha256: 32deb1126005e08991c1e6154eb2fd6538a4826a86506263f7d51eb8d0811e24
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - yp-data = yamlprocessor.dataprocess:main
+    - yp-schema = yamlprocessor.schemaprocess:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
```

```suggestion
    - python >=3.7
    - pytest-runner
```
Ref: https://github.com/JCSDA-internal/yamlprocessor/blob/1a323c0cad28bf47e2ed200e24ad5e9239a70805/setup.cfg#L40